src/Sylius/Behat/Context/Api/Admin/ManagingShipmentsContext.php

```diff
@@ -155,7 +155,7 @@ public function iShipTheShipmentOfOrderWithTrackingCode(OrderInterface $order, s
      */
     public function iShouldBeNotifiedThatTheShipmentHasBeenSuccessfullyShipped(): void
     {
-        Assert::same($this->responseChecker->getValue($this->client->getLastResponse(), 'state'), 'shipped', 'Shipment is not shipped');
+        Assert::same($this->responseChecker->isAccepted($this->client->getLastResponse()), true);
```

Comment:
```suggestion
        Assert::true(
            $this->responseChecker->isAccepted($this->client->getLastResponse()),
            'expected blablabla',
        );
```

Guessing we could throw all those `isAccepted`/`isCreationSuccessful`/etc into a generic `NotificationContext` since I assume, they'll almost always have the same code so it's unnecessary duplication.

Not in this PR though.